src/main/java/org/hisp/dhis/tasks/GetAndUpdateEventsTask.java

```diff
@@ -16,9 +16,10 @@ public GetAndUpdateEventsTask( int weight, String eventsQuery )
         this.query = eventsQuery;
     }
 
+    @Override
     public String getName()
     {
-        return "Get and update events task";
+        return "POST events";
```

Comment:
Clashing with another test

Reply:
Ah, I must have changed it when I was trying to debug the weird error from last week, reverted.